test/jdk/java/awt/print/PrinterJob/PrintAllFonts.java

```diff
@@ -33,27 +33,25 @@
  * @test
  * @bug 4884389 7183516
  * @key printer
- * @library /test/lib /java/awt/regtesthelpers
+ * @library /java/awt/regtesthelpers
  * @build PassFailJFrame
  * @summary Font specified with face name loses style on printing
  * @run main/manual PrintAllFonts
  */
 public class PrintAllFonts implements Printable {
-
-    private static Font[] allFonts;
-    private int lineHeight = 18;
+    private static final int LINE_HEIGHT = 18;
+    private static final int FONT_SIZE = 14;
+    private final Font[] allFonts =
+            GraphicsEnvironment.getLocalGraphicsEnvironment().getAllFonts();
```

Comment:
I suggest adding blank lines to visually separate fields / constants with different purpose.
```suggestion
    private static final int LINE_HEIGHT = 18;
    private static final int FONT_SIZE = 14;

    private final Font[] allFonts =
            GraphicsEnvironment.getLocalGraphicsEnvironment().getAllFonts();

```

Reply:
What about this?